src/components/forms/EditGrid/EditGridItem.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import type {Meta, StoryObj} from '@storybook/react';
 import {expect, fn, userEvent, within} from '@storybook/test';
 import {Paragraph} from '@utrecht/component-library-react';
+import '@utrecht/components/paragraph';
```

Comment:
added to ensure that the CSS is loaded for this story, since the Paragraph component is only used here and not in the component itself.